submissions/OgeEjim/DOM/signupval.js

```diff
@@ -0,0 +1,33 @@
+let submitSign = document.getElementById('validateSignup');
```

Comment:
Kindly delete this task as it is not part of newsletter form validation task

Reply:
I have deleted it ma